apps/frontend/src/libs/components/confirmation-modal/confirmation-modal.tsx

```diff
@@ -4,13 +4,15 @@ import { useCallback } from "~/libs/hooks/hooks.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	confirmLabel?: string;
```

Comment:
I think for now we can just use "Confirm" in such modals for all cases (agree with qa first).
```suggestion
```